amuletml.cabal

```diff
@@ -14,14 +14,18 @@ executable amuletml
   main-is:             Main.hs
   build-depends:       mtl >= 2.2 && < 2.3
                      , syb >= 0.7 && < 0.8
-                     , base >= 4.9 && < 4.11
                      , text >= 1.2 && < 1.3
+                     , array >= 0.5 && < 0.6
+                     , bytestring >= 0.10.8.2 && < 0.11
+                     , base >= 4.9 && < 4.11
                      , lens >= 4.15 && < 4.16
                      , parsec >= 3.1 && < 3.2
                      , monad-gen >= 0.3 && < 0.4
                      , containers >= 0.5 && < 0.6
                      , transformers >= 0.5 && < 0.6
                      , pretty-show
+  build-tools:         happy, alex
+  other-modules:       Parser.HParser, Parser.ALexer
```

Comment:
I would prefer if these names were less silly. Have you removed the Parsec ones? Can you kill the Parsec dependency?